akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -117,6 +133,10 @@ class CircuitBreaker(
     this(scheduler, maxFailures, callTimeout, resetTimeout, 36500.days, 1.0)(executor)
   }
 
+  def this(executor: ExecutionContext, scheduler: Scheduler, maxFailures: Int, callTimeout: java.time.Duration, resetTimeout: java.time.Duration) = {
+    this(scheduler, maxFailures, callTimeout.asScala, resetTimeout.asScala, 36500.days, 1.0)(executor)
```

Comment:
^

Reply:
ok, it is also an option and i will do it